torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -18,22 +18,40 @@ void confirmPendingUser(
   if (!futureMessage.hasError()) {
     auto rr = RemoteRet::fromMessage(futureMessage.constValue());
     TORCH_INTERNAL_ASSERT(rr->forkId() == expectedForkId);
+  } else {
+    // Handle errors, such as timeouts, by invoking the error handler on the
```

Comment:
Would be be correct if I assume this is best-effort error handling? As this callback is only run when receiving the response, but before that, the applications can still fork and use this UserRRef? If yes, let's add this to comment and docs accrodingly.

Reply:
Yes, will update docs to indicate this.

Reply:
@mrshenli I updated this comment to include a note. Do you have a suggestion on the best place in docs to add this note? Should it be in the docstrings for `rpc.remote`?

Reply:
@mrshenli I decided to add it in a docstring as `warning` under def remote. Let me know if something else might be better.

Reply:
This LGTM! Thanks!